cmake/DDCConfig.cmake.in

```diff
@@ -6,10 +6,20 @@
 
 include(CMakeFindDependencyMacro)
 
+find_package(Kokkos 4.4...4.5 CONFIG)
+
 if(@DDC_BUILD_KERNELS_FFT@)
    find_dependency(KokkosFFT 0.2.1...<1 CONFIG)
 endif()
 
+if(@DDC_BUILD_KERNELS_SPLINES@)
+   find_dependency(Ginkgo 1.8.0 EXACT)
+   list(PREPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
+   find_dependency(LAPACKE)
+   list(POP_FRONT CMAKE_MODULE_PATH)
```

Comment:
I thought I could use an option of `find_dependency` to search for the file FindLAPACKE.cmake, but I could not make it work.

Reply:
If the user has a `LAPACKEConfig.cmake` or its own `FindLAPACKAGE.cmake` you are bypassing it.
If you APPEND, you will give priority to the user, but it might be not compatible.
Not sure what is the good approach here.

At least you should add a comment on why it is done like this.

Reply:
I don't know either. Let's give it a try then.

I will add a comment.

Reply:
Note that we do something similar in KokkosFFT

Reply:
done in 47ada334af847d5288ab2366adb9f1e56dab2e33